pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1837,7 +1864,7 @@ func TestNodeShouldRunDaemonPod(t *testing.T) {
 			},
 			ds: &apps.DaemonSet{
 				Spec: apps.DaemonSetSpec{
-					Selector: &metav1.LabelSelector{MatchLabels: simpleDaemonSetLabel},
+					//Selector: &metav1.LabelSelector{MatchLabels: simpleDaemonSetLabel},
```

Comment:
Why is this commented out? If not needed, you should delete it.

Reply:
Forgot to add it back. I am doing it now. Thanks :)